src/simplewallet/simplewallet.cpp

```diff
@@ -718,22 +737,21 @@ bool simple_wallet::seed_set_language(const std::vector<std::string> &args/* = s
     fail_msg_writer() << tr("wallet is watch-only and has no seed");
     return true;
   }
+
+  SCOPED_WALLET_UNLOCK();
+
   if (!m_wallet->is_deterministic())
   {
     fail_msg_writer() << tr("wallet is non-deterministic and has no seed");
     return true;
   }
- 
-  const auto pwd_container = get_and_verify_password();
-  if (pwd_container)
-  {
-    std::string mnemonic_language = get_mnemonic_language();
-    if (mnemonic_language.empty())
-      return true;
 
-    m_wallet->set_seed_language(std::move(mnemonic_language));
-    m_wallet->rewrite(m_wallet_file, pwd_container->password());
-  }
+  std::string mnemonic_language = get_mnemonic_language();
+  if (mnemonic_language.empty())
+    return true;
+
+  m_wallet->set_seed_language(std::move(mnemonic_language));
+  m_wallet->rewrite(m_wallet_file, pwd_container->password());
```

Comment:
This will crash when `ask-password` is disabled where `pwd_container` would be `boost::none`. Password input is mandatory here.